### PR TITLE
Remove unused assets

### PR DESCRIPTION
**Pull request by queengooborg**
This PR removes unused assets as found by `yarn filecheck`.

### CODE REVIEW

**Comment by yin1999** [comment]
Hi @queengooborg, should we always check assets under the folders that contains changed documents in PR test.

File: files/zh-cn/web/css/css_backgrounds_and_borders/resizing_background_images/ss2.png
Comment (yin1999) [inline files/zh-cn/web/css/css_backgrounds_and_borders/resizing_background_images/ss2.png:1]: It seems that the [`firefox_logo.png`](https://github.com/mdn/translated-content/blob/881e54e12075b75078d992cae56efd1c83de0e75/files/zh-cn/web/css/css_backgrounds_and_borders/resizing_background_images/firefox_logo.png) under this folder is also not used anymore. Did filecheck tool misses this?

**Review by saionaro — APPROVED** [review]
@queengooborg seems good for RU, thank you!

**Review by hochan222 — APPROVED** [review]
LGTM for `ko`. Thank you have a nice day :)

### for yari-content-ko team
translated-content repo에서 모든 이미지 파일들은 이제 추가하지 않아도 됩니다. 참고 부탁드립니다. ( [참고](https://github.com/mdn/translated-content/pull/5031#issuecomment-1642391891) ) (cc. @mdn/yari-content-ko)

**Review by yin1999 — APPROVED** [review]
I've removed one more unused image in zh-cn folder. Thank you @queengooborg

**Review by josielrocha — APPROVED** [review]
LGTM for `pt-BR`

**Pull request merged**